packages/fxa-settings/src/pages/ResetPasswordConfirmed/index.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { usePageViewEvent } from '../../lib/metrics';
+
+type ResetPasswordConfirmedProps = {
+  serviceName: String;
+};
+
+// eslint-disable-next-line no-empty-pattern
+const ResetPasswordConfirmed = ({
+  serviceName,
+}: ResetPasswordConfirmedProps & RouteComponentProps) => {
+  usePageViewEvent('pw-reset-confirmed.confirmation');
```

Comment:
I think the new event(s) should also be added to `fxa-content-server/server/lib/amplitude.js` , following the model for the `CreatePassword` page. Could be:
```
  // Password reset confirmation
  'screen.settings.confirm-password-reset': {
    group: GROUPS.settings,
    event: 'confirm_password_reset_view',
  },
``` 
If this pattern looks right, then  the page view event here would become
`usePageViewEvent('settings.confirm-password-reset')`

Reply:
oh thank you!!

Reply:
I feel like long-term we're going to want to move away from `settings` as the group, but since we haven't actually settled how metrics are going to change, I think this is a viable way to move forward